### PR TITLE
PMM-5563 Add API tests for annotations.

### PR DESCRIPTION
**Pull request by JiriCtvrtka**

### CODE REVIEW

File: management/annotation_test.go [hunk 1]
```diff
@@ -35,4 +42,157 @@ func TestAddAnnotation(t *testing.T) {
...
+			},
+		}
+		addMySQLOK, err := client.Default.MySQL.AddMySQL(params)
+		require.NoError(t, err)
+		require.NotNil(t, addMySQLOK)
+		require.NotNil(t, addMySQLOK.Payload.Service)
+		serviceID := addMySQLOK.Payload.Service.ServiceID
+		defer pmmapitests.RemoveServices(t, serviceID)
+
+		// Check that service is created and its fields.
```
[the remark this answers is not in the document]
Comment (BupycHuk) [reply management/annotation_test.go:115]: The same in other places where we use management API.

[the remark this answers is not in the document]
Comment (JiriCtvrtka) [reply management/annotation_test.go]: Ok, it should be fixed.

File: management/annotation_test.go [hunk 2]
```diff
@@ -35,4 +39,104 @@ func TestAddAnnotation(t *testing.T) {
...
+				Port:        3306,
+				ServiceName: serviceName,
+			},
+			Context: pmmapitests.Context,
+		}
+		resService, err := inventoryClient.Default.Services.AddMySQLService(paramsService)
+		assert.NoError(t, err)
+		require.NotNil(t, resService)
+		serviceID := resService.Payload.Mysql.ServiceID
+		assert.Equal(t, &services.AddMySQLServiceOK{
```
Comment (BupycHuk) [inline management/annotation_test.go:95]: No reason to check this in current test

**Comment by JiriCtvrtka** [comment]
After PMM-5800 I should have time for this.

**Comment by BupycHuk** [comment]
@JiriCtvrtka let's update FB and add this branch to FB, wait for @askomorokhov review for this PR, check that tests are not failing in FB and move to QA.

**Pull request closed without merging**